src/GameFace/Models/Achievements.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Threading.Tasks;
+
+namespace GameFace.Models
+{
+    public class Achieve
+    {
+        [DataMember]
```

Comment:
I think we can get rid of the DataMember attribute, but I'll leave it up to you to decide - if we can't, then we need to add it for Rewards, StatsDatapoint etc.